tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -801,6 +828,72 @@ int main(int argc, char **argv)
         };
     };
 
+    /* s2n_server_nst_send */
+    {
+        /* TLS 1.2 server sends a new session ticket key */
+        {
+            DEFER_CLEANUP(struct s2n_config *config = s2n_config_new(), s2n_config_ptr_free);
+            EXPECT_NOT_NULL(config);
+            DEFER_CLEANUP(struct s2n_connection *conn = s2n_connection_new(S2N_SERVER),
+                    s2n_connection_ptr_free);
+            EXPECT_NOT_NULL(conn);
+            EXPECT_OK(s2n_resumption_test_ticket_key_setup(config));
+            EXPECT_SUCCESS(s2n_connection_set_config(conn, config));
+
+            EXPECT_NOT_EQUAL(s2n_stuffer_space_remaining(&conn->handshake.io), 0);
+
+            conn->config->use_tickets = 1;
+            conn->session_ticket_status = S2N_NEW_TICKET;
+
+            EXPECT_SUCCESS(s2n_server_nst_send(conn));
+            EXPECT_TICKETS_SENT(conn, 1);
+
+            uint32_t lifetime = 0;
+            EXPECT_SUCCESS(s2n_stuffer_read_uint32(&conn->handshake.io, &lifetime));
+            EXPECT_TRUE(lifetime > 0);
+
+            uint16_t ticket_len = 0;
+            EXPECT_SUCCESS(s2n_stuffer_read_uint16(&conn->handshake.io, &ticket_len));
+            EXPECT_TRUE(ticket_len > 0);
+        };
+
+        /* TLS 1.2 server sends zero ticket lifetime and zero-length ticket if key expires */
+        {
```

Comment:
Given that you added a compliance comment on your fix, you can also add a compliance comment on your test, which lists what line you're testing. Something like: 

```
     /*= https://www.rfc-editor.org/rfc/rfc5077#section-3.3
      *= type=test
      *# If the server determines that it does not want to include a
      *# ticket after it has included the SessionTicket extension in the
      *# ServerHello, then it sends a zero-length ticket in the
      *# NewSessionTicket handshake message.
      */
```